### PR TITLE
Join by telephone numbers are correct

### PR DESCRIPTION
**Pull request by Venkatbac**
**Before creating a pull request make sure that:**

- [ ] commit messages are meaningful and follow good commit message guidelines
- [ ] README and other documentation has been updated / added (if needed)
- [ ] tests have been updated / new tests has been added (if needed)

Please remove this line and everything above and fill the following sections:

### Change description ###

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[ ] No
```

### CODE REVIEW

File: UI/Selenium/Pages/BookingListPage.cs [hunk 1]
```diff
@@ -14,5 +14,9 @@ public class BookingListPage
         public static By HearingDetailsRowSpecific(string text) => By.XPath($"//div[text()[contains(.,'{text}')]]");
         public static By SearchCaseTextBox = By.Id("caseNumber");
         public static By SearchButton => By.Id("searchButton");
+        public static By ConfirmedButton => By.XPath("//*[contains(text(),'Confirmed')]");
```
Comment (MichaelKeaneathmcts) [inline UI/Selenium/Pages/BookingListPage.cs:17]: id is preferred

File: UI/Selenium/Pages/BookingListPage.cs [hunk 2]
```diff
@@ -14,5 +14,9 @@ public class BookingListPage
...
+        public static By TelephoneParticipantLink => By.XPath("//div[@id='conference_phone_details']");
```
Comment (MichaelKeaneathmcts) [inline UI/Selenium/Pages/BookingListPage.cs:18]: Why not By.id  ?

**Review by MichaelKeaneathmcts — CHANGES_REQUESTED** [review]

File: UI/Selenium/Pages/BookingListPage.cs (hunk 1 above)
Comment (Venkatbac) [reply UI/Selenium/Pages/BookingListPage.cs:18]: Tried With id was failing in few occasions and Xpath is working fine.

File: UI/Selenium/Pages/BookingListPage.cs (hunk 2 above)
Comment (Venkatbac) [reply UI/Selenium/Pages/BookingListPage.cs:19]: Tried With id was failing in few occasions and Xpath is working fine.

File: UI/Selenium/Steps/BookingListSteps.cs [hunk 3]
```diff
@@ -75,5 +77,42 @@ public void ThenTheBookingShouldContainExpectedValues()
...
+        {
+            ExtensionMethods.FindElementWithWait(Driver, BookingListPage.ConfirmedButton, _scenarioContext).Click();
+            ExtensionMethods.FindElementWithWait(Driver, BookingListPage.TelephoneParticipantLink, _scenarioContext).Click();
+            _hearing.BookingList.TelephoneParticipantLink = new TextCopy.Clipboard().GetText();
+        }
+
+        [Then(@"telephone participant link should be copied")]
+        public void ThenTelephoneParticipantLinkShouldBeCopied()
+        {
+            Assert.IsTrue(_hearing.BookingList.TelephoneParticipantLink.Contains("+448000488500"), "Phone verified");
```
[the remark this answers is not in the document]
Comment (MichaelKeaneathmcts) [reply UI/Selenium/Steps/BookingListSteps.cs:130]: Moved to be a cleanup task

**Review by MichaelKeaneathmcts — APPROVED** [review]

**Review by MichaelKeaneathmcts — APPROVED** [review]

**Pull request merged**